src/routes/safe/components/Transactions/TxList/TxSummary.tsx

```diff
@@ -33,54 +89,28 @@ export const TxSummary = ({ txDetails }: Props): ReactElement => {
           <TxShareButton id={txDetails.txId} />
         </div>
       )}
-      <div className="tx-hash">
-        <Text size="xl" strong as="span">
-          Transaction hash:{' '}
-        </Text>
-        {txHash ? (
-          <InlineEthHashInfo textSize="xl" hash={txHash} shortenHash={8} showCopyBtn explorerUrl={explorerUrl} />
-        ) : (
-          <Text size="xl" as="span">
-            {NOT_AVAILABLE}
-          </Text>
-        )}
-      </div>
-      {safeTxHash !== undefined && (
-        <div className="tx-hash">
-          <Text size="xl" strong as="span">
-            SafeTxHash:{' '}
-          </Text>
-          <InlineEthHashInfo textSize="xl" hash={safeTxHash} shortenHash={8} showCopyBtn />
-        </div>
-      )}
-      {nonce !== undefined && (
-        <div className="tx-nonce">
-          <Text size="xl" strong as="span">
-            Nonce:{' '}
-          </Text>
-          <Text size="xl" as="span">
-            {nonce}
-          </Text>
-        </div>
+      {txHash && <TxDataRow title="Transaction hash:" value={txHash} inlineType="hash" />}
+      {safeTxHash && <TxDataRow title="SafeTxHash:" value={safeTxHash} inlineType="hash" />}
+      {created && <TxDataRow title="Created:" value={formatDateTime(created)} />}
+      <TxDataRow title="Executed:" value={executedAt ? formatDateTime(executedAt) : NOT_AVAILABLE} />
+      <br></br>
+
+      {/* Advanced TxData */}
+      {txData?.operation !== undefined && (
+        <TxDataRow title="Operation:" value={`${txData?.operation} (${Operation[txData?.operation].toLowerCase()})`} />
       )}
-      {created && (
-        <div className="tx-created">
-          <Text size="xl" strong as="span">
-            Created:{' '}
-          </Text>
-          <Text size="xl" as="span">
-            {formatDateTime(created)}
-          </Text>
-        </div>
+      {safeTxGas && <TxDataRow title="safeTxGas:" value={safeTxGas} />}
```

Comment:
Maybe you should use proper terms instead of keys here, e.g. 'Gas price:'

Reply:
Let's comment it on the ticket so Designs can reflect it.

Reply:
Why can't you just adjust the text here? It's a label.

Reply:
I understand it is an easy tweak. My point is there are more fields where we are using the keys. For now it is coherent with the Mobile implementation reason why I think should have Johannes helping with the renaming and addressing it in both applications (if applicable).

Reply:
I follow you, but some of the keys are just camel cased labels. If you'd sooner shift it to the design ticket, fair enough.